Dockerfile

```diff
@@ -16,12 +25,16 @@ COPY controllers/ controllers/
 COPY pkg/ pkg/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+RUN WASM_SHIM_SHA256=$(cat /opt/kuadrant/wasm-shim/kuadrant-ratelimit-wasm.sha256) \
```

Comment:
In the case we follow what Gui suggested, this var makes sense... if not, probably something like `RATELIMIT_SHIM_SHA256` would make more sense

Reply:
keeping as `WASM_SHIM_SHA256`